lib/util.js

```diff
@@ -577,25 +577,61 @@ const util = {
     progressLog('Done updating submodules...')
   },
 
-  gclientSync: (reset = false, options = {}) => {
-    let args = ['sync', '--force', '--nohooks', '--with_tags', '--reset',
-                '--with_branch_heads', '--upstream',
-                '--revision', 'src/brave@' + config.getProjectRef('brave-core')]
-    if (reset)
-      args = args.concat(['--revision', 'src@' + config.getProjectRef('chrome')])
+  checkoutBraveCore: (options = {}) => {
```

Comment:
this is for future use of moving most of the brave-core scripts into brave-browser, but it also prevents brave-core from using git cache which is helpful when doing git fetch/pull